Specs/Scene/PointCloud3DTileContentSpec.js

```diff
@@ -1155,7 +1155,7 @@ describe(
           tileset.clippingPlanes = new ClippingPlaneCollection({
             planes: [
               new ClippingPlane(Cartesian3.UNIT_Z, -10.0),
-              new ClippingPlane(Cartesian3.UNIT_X, 0.0),
+              new ClippingPlane(Cartesian3.UNIT_X, 1.0),
```

Comment:
This test have the clipping plane close to the center. It can generate a different color. I move it off a little bit. This is the [sandcastle example](http://localhost:8080/Apps/Sandcastle/index.html#c=nVRtb9owEP4rJ740qMyhq/qF0modbadJXYeATtPGNJnkgOscO7IdWjrx33d5o6FvHyaCHNt3z12e57HDcGhIexgok8UwW8PQmrmAIXq0cKYU6g4MjMqSGUm40bRC68ivYWRmxlPk4ErOBIwjqTXpRQ5wpvBW6tgamFgTRUYR8BS+SI8EAzKRtIpQTPVKWlgR3nGhE9B4BwN0lCXiW7EWTFtRMR8Y7SVptNNWB/5ONcBCmRn2YC6Vw85Ub9rHU13CeVLo0O/ilcPh+aTcDAqMzKoeTFtChOUzTjFy4bn0MtyJL+kp2Gm8jj59DKta4tYZza1t+6jXLcp4zWwm5FD4JepgnunIk9FBu2ihDowUpSmTN1RSo3vSe3OPdVBYIhQAAGmR0oOf5RReTQ3qRWk9v0l9KG6uP09+/+jAu4Ou6LY7/4vwvQON/F/VmJgYFUtu6b5Xw02s1G5ubOIESuevjfXLm3RiLuke40srEwzqJmpqZibTMXcwTpdoUUSo2ZZlUF0x08xH3ekIFzxzj94AKFSBymjCMQSK1FJCnr3shIzjoKq2E/hgTDIxu1v5n/uHQLHFiIXqHvPQf0VIwWdn4Zccsr/fhkqwiLszCoUyi+CVtAVblNplRYC8VKHyBRuLz91JjcTiGz6J/Lm9t+naKuvXKZ+aveEj2t52ryjRe8QGiClB7Uo2m56o5X8fHIluB46a3mHd+ZRbkmorOnvWWDG6OBd35JdnKl3KoCsOmjlV7TfpaISbzCsqEmyGz9eLgs/L13Gb8mVT0VupjTkdVNmhQXepwmZ7tvNfq9PqO79WeFpjfqAkZTPnd0rAV4nHhCGYpXCWRX/yL3KulrMfNlP7Ma2A4pMXrjqIlHSOd+aZUmN6wGnrtB9y/LNUZWSu+Ve+mZVc52HLg9OrclEI0Q95+nKmN0bNpH2C/A8) for the test

Reply:
Hm, not sure I understand what to look out for here

Reply:
The test fails on my machine if the distance is 0.0. I tried on Sandcastle and it seems to work the same.